<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CircularProgressSettings</class>
 <widget class="QWidget" name="CircularProgressSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="3">
    <widget class="MaterialCircularProgress" name="progress"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="progressTypeLabel">
     <property name="text">
      <string>Progress type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="progressTypeComboBox">
     <item>
      <property name="text">
       <string>Determinate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Indeterminate</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="progressLabel">
     <property name="text">
      <string>Progress</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="lineWidthDoubleSpinBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lineWidthLabel">
     <property name="text">
      <string>Line width</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="progressSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="sizeSpinBox">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="disabledCheckBox">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialCircularProgress</class>
   <extends>QProgressBar</extends>
   <header location="global">QtMaterialWidgets/circularprogress.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
